### PR TITLE
chore: Use new layout colours and figma re-design - Step 4

### PR DESCRIPTION
**Pull request by lilyndk**
## Description

 Interior pages: incorporate new colours, type variables, and html page wrapper structure

NOTE: To avoid review of long PRs, this PR is dependent on/'split from' the [main branding  change PR](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/pull/195), i.e. dependent on the changes in:
- [layout/default.html](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/pull/195/files#diff-907a69846a1f6b238f1c43199984197d12c7eab26f3c3adcd45d628b26644950)
-  [custom/_variables.scss](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/pull/195/files#diff-1c370c0b2176f6f6eb6212e2f11886088d15871f9016c8b29428063bc5434b10)
- [main.scss](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/pull/195/files#diff-28cfd3e1ba32d3c3a10559c84da88a312ddd3c22851b55a8b255e424fc614eed)
- [custom.scss](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/pull/195/files#diff-763ac3523acb81380882833d64791adae805b37ea3429ea95cb44b8f5c439593)
- [mixins.scss](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/pull/195/files#diff-4dbddf5e69008e6530690782987530e639bfd5d8497cf59abcbedbc4d4aa3576)
- [page-header.html](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/pull/195/files#diff-7c00ab81dd0968f6fb395b5986f71fd56704c48a71f0b9583058f5aaaad03aa7)

New branding
- Added new colour and typography variables

Bug/refactor
- Fixed inconsistent page HTML structure and related inconsistent vertical spacings
- Refactored pages to use standardised page markup:
Screenshot 2024-05-10 at 16 07 14

## Change Type
- [X] Bug Fix
- [X] New Feature
- [X] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [ ] Other

## Related Issue
- Update branding colour: https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/189
- Create reusable font styles: https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/38
- bug: Fix inconsistent page use of semantic HTML elements (https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/200)

## Screenshots
AFTER:
program-study-group
BEFORE:
screencapture-womencodingcommunity-program-study-group-2024-05-09-21_36_51

AFTER:
book-club
BEFORE:
screencapture-womencodingcommunity-program-book-club-2024-05-09-21_36_34

AFTER:
wcc-code-of-conduct
BEFORE:
screencapture-womencodingcommunity-code-of-conduct-2024-05-10-16_32_54

AFTER:
about
BEFORE:
screencapture-womencodingcommunity-about-2024-05-09-21_37_06

AFTER:
code-of-conduct
BEFORE:
screencapture-womencodingcommunity-mentorship-code-of-conduct-2024-05-10-16_36_02

AFTER:
interview-prep
BEFORE:
screencapture-womencodingcommunity-program-interview-preparation-2024-05-09-23_34_32

AFTER:
## Pull request checklist
faq
BEFORE:
screencapture-womencodingcommunity-faq-2024-05-09-21_36_23

AFTER:
Screenshot 2024-05-09 at 21 32 23
BEFORE:
Screenshot 2024-05-09 at 21 35 43

Please check if your PR fulfills the following requirements:
- [X] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [X] I have tested my changes locally.
- [X] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
-                            {% include icons/plus.svg %}
+<div class="page faq">
+    {% include page-header.html %}
+    <div class="container">
+        <div class="row">
+            <div class="content-constrained">
+                <section>
+                    {% for faq in site.data.faq %}
+                    <div class="accordion">
+                        <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-{{faq.index}}" aria-expanded="false" data-toggle="collapse" href="#faq-{{faq.index}}" role="button">
```
Comment (lilyndk) [inline faq.html]: In a follow up PR after this is merged, we should reduce all these style classes and add styles in the new _accordion.scss file instead.

File: _sass/custom/_accordion.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.content-constrained {
```
Comment (dricazenck) [inline _sass/custom/_accordion.scss:1]: Please import this file again, because I had to remove from the previous pull request.

File: about.html [hunk 3]
```diff
@@ -1,49 +1,51 @@
 ---
 layout: default
-title: About
+title: About Us
```
Comment (dricazenck) [inline about.html:3]: 👍

File: events.html [hunk 4]
```diff
@@ -1,17 +1,19 @@
...
-    <h1>Online and In-Person Events</h1>
-</div>
-<div class="container">
-    <div class="content-constrained">
-        <div class="section">
-            <p>Women Coding Community events and meetups present great opportunities to improve your skill set, deepen your understanding of various topics, expand your professional network, and engage in meaningful social interactions.</p>
-            <p>Our community hosts various engaging events, including webinars, hands-on technical workshops, study groups, panel discussions and keynotes featuring industrial experts and tech professionals. Whether online or in-person, these gatherings provide an invaluable opportunity to share both professional insights and personal experiences with fellow members. Through collaborative learning and networking, participants can broaden their horizons, stay abreast of the latest trends, and exchange knowledge within the community.</p>
-            <p>Join us on Slack in <a href="https://womencodingcommunity.slack.com/archives/C0700RUUTJR" target="_blank">#Events</a> channel to stay up to date with the upcoming events.</p>
+<div class="page events">
+    {% include page-header.html %}
```
Comment (dricazenck) [inline events.html:7]: now I am wondering if would be better to had added this as part of the layout file, instead been adding in all pages 🙈

File: faq.html [hunk 5]
```diff
@@ -1,32 +1,33 @@
 ---
 layout: default
-title: FAQ
+title: Mentorship FAQ
```
Comment (dricazenck) [inline faq.html:3]: 👍

File: program-interview-preparation.html [hunk 6]
```diff
@@ -3,135 +3,137 @@
...
+                    <p>Are you ready to take your CV to the next level? Join us for an exciting opportunity to elevate your
+                        professional profile with our CV Clinic! 🚀</p>
+                    <p>At the Women Coding Community, we believe in the power of collaboration and support. That's why we're
+                        thrilled to offer our members a unique chance to receive personalized feedback and guidance on their CVs
+                        from their peers.</p>
+                    <p>Our CV Clinic is designed to provide you with valuable insights, expert tips, and constructive feedback
+                        to help you craft a standout CV that showcases your true potential.</p>
+                    <div class="faq my-4">
+                        <div class="accordion" id="accordion-1">
+                            <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top"
```
Comment (dricazenck) [inline program-interview-preparation.html]: Same here about the classes for the next PR

File: program-interview-preparation.html [hunk 7]
```diff
@@ -3,135 +3,137 @@
...
+                                    <h2 class="accordion-title">Benefits of joining</h2>
+                                    <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
+                                        {% include icons/plus.svg %}
+                                    </div>
+                                </div>
+                            </a>
+                            <div class="collapse" id="interview-2">
+                                <div class="card card-body border-0 p-0">
+                                    <ul>
+                                        <li><u>Expert Feedback:</u> Receive personalized feedback and tips from peers who
```
Comment (dricazenck) [inline program-interview-preparation.html]: I guess also this could be improved later `<u>Expert Feedback:</u>`

**Review by dricazenck — APPROVED** [review]

**Pull request merged**